airflow/www/views.py

```diff
@@ -2713,7 +2699,7 @@ def legacy_tree(self):
     )
     @gzipped
     @action_logging
-    def tree(self):
+    def legacy_tree(self):
```

Comment:
If `/legacy_tree` wasn't there before, wouldn't you want to keep the redirect from `/tree`?

```suggestion
    def tree(self):
```

Reply:
We don't use `Airflow.tree` anywhere. This was to be consistent with the other `legacy_*` redirects (graph, gantt, code, etc)